Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -34,15 +36,10 @@ void NANDImporter::ImportNANDBin(const std::string& path_to_bin,
   if (!ReadNANDBin(path_to_bin, get_otp_dump_path))
     return;
 
-  const std::string nand_root = File::GetUserPath(D_WIIROOT_IDX);
-  m_nand_root_length = nand_root.length();
-  if (nand_root.back() == '/')
-    m_nand_root_length++;
-
   FindSuperblock();
-  ProcessEntry(0, nand_root);
-  ExportKeys(nand_root);
-  ExtractCertificates(nand_root);
+  ProcessEntry(0, "");
```

Comment:
Don't you need to pass m_nand_root here?

Reply:
No, because the code now thinks about the path independently from the NAND root on the real filesystem. Only when it needs to interact with the real filesystem does it prepend the `m_nand_root` to the path.